clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -401,6 +401,18 @@ protected Options getStoreOptions(StoragePartitionConfig storagePartitionConfig,
       tableConfig.setCacheIndexAndFilterBlocks(rocksDBServerConfig.isRocksDBSetCacheIndexAndFilterBlocks());
       tableConfig.setFormatVersion(rocksDBServerConfig.getBlockBaseFormatVersion());
       options.setTableFormatConfig(tableConfig);
+
+      /**
+       * Only enable blob files for block-based format.
+       */
+      if (rocksDBServerConfig.isBlobFilesEnabled()) {
+        options.setEnableBlobFiles(true);
+        options.setEnableBlobGarbageCollection(true);
```

Comment:
Is this GC useful for batch-only stores? Should it be enabled only post-EOP instead? If the overhead is null or negligible on a batch-only store then no problem in leaving it always on...

Reply:
RocksDB SST file writer doesn't respect `blob file` option as all the input data are ordered, so there is no compaction.
That means this option doesn't hurt for batch-only stores.

I am planning to add some metrics and tests to validate this new mode.